src/components/LoginButton.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
```

Comment:
There seems to be little to no  diffenerence between this button and the sign-up button. If they are the same, maybe you could get rid of one of these and just call it "Button".

Reply:
Hey Julian, thanks for the feedback. I will take a look at your suggestions and try to implement them :)  By the way, it is Cristina not Christiana :)